checker/tests/nullness/RequiresNonNullTest.java

```diff
@@ -143,4 +143,10 @@ void notHiddenTest() {
     void test() {
         o = null;
     }
+
+    @RequiresNonNull("thisShouldIssue1Error")
+    // Test case for Issue 1051
+    // https://github.com/typetools/checker-framework/issues/1051
+    //:: error: (flowexpr.parse.error)
```

Comment:
Created https://github.com/typetools/checker-framework/issues/1129 to track this.